tests/integration/test_transcribe.py

```diff
@@ -60,6 +60,46 @@ def is_transcription_done():
 
         snapshot.match("GetError", e_info.value.response)
 
+    @pytest.mark.skip_offline
+    @pytest.mark.aws_validated
+    @pytest.mark.skip_snapshot_verify(
+        paths=[
+            "$..TranscriptionJob..MediaSampleRateHertz",
+            "$..TranscriptionJob..Settings",
+            "$..Error..Code",
+        ]
+    )
+    def test_transcribe_support_variable_formats(
+        self, transcribe_client, transcribe_create_job, snapshot
+    ):
+
+        file_list = [
+            "files/en-gb.wav",
+            "files/en-gb.mp3",
+            "files/en-gb.mp4",
+            "files/en-gb.ogg",
+            "files/en-gb.webm",
+            "files/en-gb.flac",
+        ]
+
+        for file in file_list:
+            file_path = os.path.join(BASEDIR, file)
+            job_name = transcribe_create_job(audio_file=file_path)
+            transcribe_client.get_transcription_job(TranscriptionJobName=job_name)
+
+            def is_transcription_done():
```

Comment:
Since you re-used check `is_transcription_done` from the function `test_transcribe_happy_path` - instead of having duplicate code, can we refactor this into a fixture?